uktrade/data-hub-frontend
# Make email marketing opt-in for new contacts

## Description of change

When adding a new company contact, we now need to opt them into email updates from us (for data protection reasons) - this PR updates the new company form to reflect this. 

## Test instructions
Go to the companies list, and pick a company.
Click 'Company Contacts'
Click 'Add contact'
Scroll down the form, and the email checkbox should read: `The company contact does accept email marketing`
Complete the form and check the expected outcome is correct on the contact summary page.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
